sci-geosciences/josm-bin/metadata.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="project">
+    <email>sci-geosciences@gentoo.org</email>
+    <name>Gentoo Geosciences Project</name>
+  </maintainer>
+  <maintainer type="project">
+    <email>proxy-maint@gentoo.org</email>
+    <name>Proxy Maintainers</name>
+  </maintainer>
+  <maintainer type="person">
+    <email>henning@hennsch.de</email>
+    <name>Henning Schild</name>
+  </maintainer>
```

Comment:
Move proxy-maint below yourself, add proxied tags like in:
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide#Proxied_maintainer_in_metadata.xml

Not sure geosciences should still be there unless you asked.

Reply:
> Not sure geosciences should still be there unless you asked.

This is okay, I specifically asked for geosciences to be added here since we also maintain the old non-bin version.